contrib/mesos/pkg/executor/executor.go

```diff
@@ -162,39 +173,20 @@ func New(config Config) *Executor {
 		staticPodsConfigPath: config.StaticPodsConfigPath,
 		launchGracePeriod:    config.LaunchGracePeriod,
 		nodeInfos:            config.NodeInfos,
+		initCompleted:        make(chan struct{}),
 	}
+	runtime.On(k.initCompleted, k.sendLoop)
```

Comment:
happy to rename, though it was named this before :)

On Wed, Dec 9, 2015 at 11:21 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/executor/executor.go
> <https://github.com/kubernetes/kubernetes/pull/17917#discussion_r47112612>
> :
>
> >  	}
> > +	runtime.On(k.initCompleted, k.sendLoop)
>
> s/sendLoop/runSendLoop/. I guess this doesn't send a loop ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17917/files#r47112612>.
>